Source/SwiftLintFramework/Rules/VariableNameRule.swift

```diff
@@ -41,18 +53,29 @@ public struct VariableNameRule: ASTRule {
     public func validateFile(file: File, kind: SwiftDeclarationKind,
                              dictionary: [String: SourceKitRepresentable]) -> [StyleViolation] {
         return file.validateVariableName(dictionary, kind: kind).map { name, offset in
+            guard !config.excluded.contains(name) else {
```

Comment:
In general, I think `if A {}` is preferable to `guard !A else {}`... we could even write a rule for this :wink: